lib/utils.js

```diff
@@ -95,7 +102,11 @@ exports.json2CSV = function (obj) {
 
     });
 
-    return '"' + data.join('","') + '"\n';
+    if (labels.length) {
+        str = labels.join() + '\n';
+    }
+
+    return str + '"' + data.join('","') + '"\n';
```

Comment:
Please rewrite using es6 template strings, I think it would be much easier to read.